hazelcast/src/main/java/com/hazelcast/jet/JetService.java

```diff
@@ -208,6 +209,13 @@ default Job newLightJob(@Nonnull DAG dag) {
      */
     Job newLightJob(@Nonnull DAG dag, @Nonnull JobConfig config);
 
+    /**
+     * Submits a job defined in the Core API with attached {@link Subject}.
+     * <p>
+     * See {@link #newLightJob(Pipeline, JobConfig)} for more information.
+     */
+    Job newLightJob(@Nonnull DAG dag, @Nonnull JobConfig config, @Nullable Subject subject);
```

Comment:
I don't think it should be a public API:
1. It does not make sense on client side
2. even on member side it is meant to be internal way of invoking jobs

Reply:
ok

Reply:
https://github.com/hazelcast/hazelcast/pull/25408/commits/236d0551e3128366d66c0cf690ec36f1424222dc